nodeJS/orms/prisma_orm.md

```diff
@@ -0,0 +1,157 @@
+
+Writing more code is not necessarily a bad thing, and we even suggest that you explore these above ideas in your solo projects. But as you start to work in teams, and with large scale software, it becomes imperative to have a standard way of interacting with the database, either with an external library or a homegrown solution.
+
+<div class="lesson-note lesson-note--tip" markdown="1">
+
+If you haven't applied any of the above paradigms in your previous projects, we highly recommend going back and refactoring. You might end up with a very basic mini ORM for your project. This will help you appreciate a bit more how a proper ORM will make your life easier.
+
+</div>
+
+#### Navigating the codebase
+
+When all database interactions are done in raw SQL, there's nowhere in the codebase where you can understand the database tables, their relations, and column data types. You might have to log in to your database to make sense of what the codebase is doing. To attain a technical understanding of the project, you now rely on the codebase as well as access to the database.
+
+Most ORMs out there solve this problem by bringing database definitions into the codebase. This is called a "schema". This allows you to quickly glance at the schema of a table and understand what columns it has and so forth.
+
+#### Altering production data
+
+Databases inevitably change as the requirements of the project evolve. You might have to add a new column or populate a new table with existing data. In technical terms, this is called a migration. Without an ORM or a similar library, you'll be hand rolling these migrations, which will be error-prone and tedious. ORMs standardize migrations via changelogs, and have processes to deal with conflicts. Granted, you won't be running as many migrations in your curriculum projects, but as you work professionally, you might do it every other day.
+
+### Introducing Prisma ORM
+
+ORMs pretty much solve all of the pain points mentioned above, but it's not all sunshine and roses. There can be a learning curve to fully understand the ins and outs of an ORM, and some don't even fully support all SQL features. Even with these shortcomings, using an ORM can be extremely worthwhile.
+
+In the Node.js ecosystem, there are A LOT of ORMs to choose from, and the community has not landed on a go-to yet. We've decided to cover [Prisma ORM](https://www.prisma.io/) in the curriculum because of its popularity and community support.
+
+Prisma ORM has all the features we need to complete this curriculum and then some. Prisma ORM consists of several libraries, which means you can just use `npm install` to use it. We'll start by discussing some of its features, and then link to some official Prisma ORM guides to help you get started.
```

Comment:
I intended the sentence to read that "Prisma has several libraries that you can use via npm".

I don't want to show the commands here as that's best suited during the assignment readings. Do you have a suggestion on how we can reword this sentence?

Reply:
That's fair. Perhaps then something like the below?

```suggestion
Prisma ORM has all the features we need to complete this curriculum and then some. Prisma ORM consists of several libraries, so we can use npm to install whichever one(s) our application needs. We'll start by discussing some of its features, and then link to some official Prisma ORM guides to help you get started.
```

Perhaps also change the href for the link in the paragraph above (currently `https://www.prisma.io/`) to `https://www.prisma.io/orm` instead? Since the link text indicates its to the ORM directly, and not Prisma in general/its other products.